src/app/view/atoms/button/button.component.html

```diff
@@ -0,0 +1 @@
+<button (click)="buttonClick()"> {{text | uppercase}} </button>
```

Comment:
Why we create a component for button and i didn't see that we used atomic design so why folder calls atoms ?